sdk/keyvault/keyvault-keys/samples/typescript/src/cryptography.ts

```diff
@@ -29,7 +29,9 @@ export async function main(): Promise<void> {
   // Connection to Azure Key Vault Cryptography functionality
   const myWorkKey = await client.createKey(keyName, "RSA");
 
-  const cryptoClient = new CryptographyClient(myWorkKey.id!, credential);
+  const cryptoClient = new CryptographyClient(
+    myWorkKey.id! // Or just `myWorkKey`. You can use either the key or the key Id to create a CryptographyClient.
+  , credential);
```

Comment:
The changes here should match what is done in the js sample

Reply:
Done!